controllers/territory_controller/territory_controller.py

```diff
@@ -335,7 +349,10 @@ def set_territory_ownership(
             )
             return
 
-        if self._claim_log.get_claim_count(station_code) == LOCKED_OUT_AFTER_CLAIM - 1:
+        if (
+            self._claim_log.get_claim_count(station_code) == LOCKED_OUT_AFTER_CLAIM - 1 and
+            claimed_by != Claimant.UNCLAIMED
```

Comment:
Maybe also update this one?
```suggestion
            claimed_by in Claimant.zones()
```